evalml/pipelines/components/transformers/column_selectors.py

```diff
@@ -5,12 +5,15 @@
 
 
 class ColumnSelector(Transformer):
-    def __init__(self, columns=None, random_seed=0, **kwargs):
-        """Initalizes an transformer that drops specified columns in input data.
+    """
```

Comment:
Why do some lines have the quotes on one line and the description on the next, like the line here, while others are on the same line (like the description for `SelectColumns` on line 97)?

Reply:
Ah, I don't think there's a difference tbh :d